samples/DevTestLabs/SimpleUI/README.md

```diff
@@ -0,0 +1,25 @@
+﻿# Simple UI Sample Project
+
+
+## Overview
+The sample provided uses an ASP.NET Core web application with React for the user interface. 
+
+## Setup and Deployment
+### Create an App Registration for your client application in Azure Active Directory (AAD)
+Before deploying this sample you will need to create an App Registration. This App Registration represents the identity of your client application, and will enable you to authenticate with the user's credentials to various Azure services (namely, DevTest Labs in this example). 
+1. [Follow these instructions to register your application.](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) You will want to select the "Single-page application" tile when configuring platform settings.
+2. In the management panel of your app registration, select *API permissions* > *Add a permission* > *Azure Service Management*
+3. Select the *user_impersonation* permission
+4. Select *Add permissions* to complete
```

Comment:
Double check for locked down AAD tenants if there is an extra step for global aad admins to do...  If so, we should add a note for this in the readme

Reply:
It seems that, like we noted, "Admin consent required" is set to false in the Portal:
![image](https://user-images.githubusercontent.com/8703291/109185859-64667700-775e-11eb-9449-ae506c36290f.png)
However, when I try logging in via the client, I get this error:
![image](https://user-images.githubusercontent.com/8703291/109185958-7ba56480-775e-11eb-95de-81b1b4c35fdb.png)

Does this seem like the appropriate article to link to? https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/grant-admin-consent The user will need to figure out who their AAD admins are, though

Reply:
We should talk about this in our next meeting - tenant wide consent is a big hammer and there might be a less invasive approach...  Thanks!

Reply:
Sounds good -- I'll make a note of this so that I don't forget to bring it up!